### ISSUE TITLE
Could not find declaration file (TypeScript) in v2.0.16

### ISSUE THREAD

**Issue by aborile** [amplitude/Amplitude-ReactNative#67]
I believe that this library has built-in type declarations, but warning message comes out when I upgrade library to v2.0.16.
If I downgrade it to v2.0.12, the second latest version, then there is no warning about type declaration. 

```
Could not find a declaration file for module '@amplitude/react-native'. '/Users/mac/[Project root]/node_modules/@amplitude/react-native/lib/commonjs/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/amplitude__react-native` if it exists or add a new declaration (.d.ts) file containing `declare module '@amplitude/react-native';`
```

And I think any type declaration file such as `@types/amplitude__react-native` does not exist in npm.
Do I have to do something to add declaration file?

![image](https://user-images.githubusercontent.com/37267486/114657138-5e3a5300-9d2a-11eb-9b57-485cdac8b050.png)

**Comment by kelsonpw** [comment]
@aborile this have been fixed in 2.0.17 
![image](https://user-images.githubusercontent.com/17172317/114740674-eacf1c00-9cfe-11eb-9daa-f1cdc82a805a.png)

```  "dependencies": {
    "@amplitude/react-native": "^2.0.17",
    ...
  },
```

**Comment by kelsonpw** [comment]
The problem was the "types" key in package.json was pointing to the wrong file.
